### DISCUSSION TITLE
How to show X number of recent full posts on homepage?

### DISCUSSION THREAD

**Discussion by moffat** [mmistakes/minimal-mistakes#3762] [Q&A] [answered]
As the subject says, I'm looking to show X (say 3) recent full posts on the homepage, instead of a list of recent posts with links and a brief summary. Any ideas? (In case it isn't super obvious, I'm very new to Jekyll).

**Comment by lwasser** [comment, marked as answer]
hey @moffat you'd want something like: 

```html
{% for post in site.posts limit:3 %}
   {{ post.title }} 
{% endfor %}
```

Which would look through 3 posts - i think the 3 most recent posts should be displayed there. if you wanted 5 use `limit:5`

then you can add any styling that you wish around that loop to style how each is printed and linked to. Cheers!

SO i have something like that in my sites that looks like this that lists them horizontally using min mistakes styles.:

```html
<div class="feature__wrapper">
   {% for post in site.posts limit:3 %}
   <div class="feature__item">
      <div class="archive__item">
         <div class="archive__item-body">
            <h3 class="archive__item-title"><a href="{{ site.baseurl }}{{ post.url}}" rel="permalink">{{ post.title }}</a></h3>
            <div class="archive__item-excerpt">
               <p>{{ post.excerpt | markdownify }}</p>
            </div>
         </div>
      </div>
   </div>
   {% endfor %}
</div>
```

**Reply by moffat** [reply]
Thank you @lwasser!
